dji_sdk/src/modules/dji_sdk_node_main.cpp

```diff
@@ -50,10 +52,19 @@ void DJISDKNode::broadcast_callback()
     auto current_time = ros::Time::now();
 
 
+
```

Comment:
Did you check that bc_data doesn't have a receive time stamp? If it was high quality it clearly would..
If not consider moving up the `ros::Time::now()` call right below the `getBroadcastData()` call.